packages/core/src/components/screens/language/components/SelectLanguageCellGroup.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Cell, Typography, Radio} from "@equinor/mad-components";
+import { useLanguage } from "../../../../store/language"
+
+type SelectLanguageCellGroupProps = {selectedLanguageCodeDraft: string, setSelectedLanguageCodeDraft: (code: string) => void}
+
+export const SelectLanguageCellGroup = ({selectedLanguageCodeDraft, setSelectedLanguageCodeDraft}: SelectLanguageCellGroupProps) => {
+    const language = useLanguage();
+    const supportedLanguages = language.getSupportedLanguages();
+    const onPress = (code: string) => () => setSelectedLanguageCodeDraft(code)
+    return <Cell.Group>
+        {supportedLanguages.map(language => 
+            <Cell key={language.code} rightAdornment={<Radio checked={language.code === selectedLanguageCodeDraft} onPress={onPress(language.code)} />} onPress={onPress(language.code)}>
```

Comment:
So if we make radio's pressable optional, it would make the press flow through to the cell pressable? In that case I agree. I added it to both now to make sure the press was registered no matter where they press

Reply:
I think its enough with press on the whole cell

Reply:
Fixed